packages/seal-loupe-adapter/composer.json

```diff
@@ -26,13 +26,10 @@
             "email": "alexander@sulu.io"
         }
     ],
-    "conflict": {
-        "doctrine/dbal": "<3.6.0"
-    },
```

Comment:
conflict not longer required as dbal version already defined correctly now for loupe